drv/cosmo-hf/src/main.rs

```diff
@@ -60,18 +60,29 @@ fn main() -> ! {
     userlib::hl::sleep_for(1000);
 
     let id = unsafe { reg::BASE.read_volatile() };
-    assert_eq!(id, 0x1de);
+    if id != 0x1de {
+        fail(drv_hf_api::HfError::FpgaNotConfigured);
```

Comment:
What's the semantics of this error? Is it possible that the FPGA could later be configured, and the host-flash driver will need to be kicked back into actually working? Or is that a problem to solve later?

Reply:
I'm putting that into the "problems to be solved later" bucket!

Reply:
Cool, carry on then!